spec/mail/notify/mailer_spec.rb

```diff
@@ -211,9 +211,15 @@
       end
     end
 
-    context "when given a blank value" do
+    context "when given an empty string" do
       it "returns explicitly blank personalisation" do
```

Comment:
Should this be `it "returns an empty string" do` per below?

Reply:
It could be.